app/models/multilingual/custom_translation.rb

```diff
@@ -1,48 +1,55 @@
 # frozen_string_literal: true
-class Multilingual::TranslationFile
-  include ActiveModel::Serialization
+class Multilingual::CustomTranslation < ActiveRecord::Base
+  self.table_name = 'custom_translations'
 
   PATH ||= "#{Multilingual::PLUGIN_PATH}/config/translations".freeze
   KEY ||= 'file'.freeze
 
-  attr_accessor :code, :type
-
-  def initialize(opts)
-    @code = opts[:code].to_sym
-    @type = opts[:type].to_sym
-  end
+  validates :file_name, :code, :file_type, :file_ext, :translation_data, presence: true
+  serialize :translation_data
+  before_save :save_file
+  after_save :after_save
```

Comment:
This is redudant. You can just do
```
after_save do
 # logic
end
```

Reply:
Oh nice, I'll delete after_save

Reply:
@angus, this fails tests after removal, doesn't seem redundant to me?